arangod/Cluster/ReplicationTimeoutFeature.cpp

```diff
@@ -44,10 +56,15 @@ ReplicationTimeoutFeature::ReplicationTimeoutFeature(application_features::Appli
 void ReplicationTimeoutFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
   options->addSection("cluster", "Configure the cluster");
 
-  options->addOption(
-      "--cluster.synchronous-replication-timeout-minimum",
-      "all synchronous replication timeouts will be at least this value (in seconds)",
-      new DoubleParameter(&lowerLimit));
+  options->addOption("--cluster.synchronous-replication-timeout-minimum",
+                     "all synchronous replication timeouts will be at least "
+                     "this value (in seconds)",
+                     new DoubleParameter(&lowerLimit));
+
+  options->addOption("--cluster.synchronous-replication-timeout-maximum",
```

Comment:
There doesn't seem to be a validation in place via `validateOptions(...)` that `lowerLimit` <= `upperLimit`

Reply:
Correct, this needs fixing.